client/ayon_core/hosts/houdini/plugins/create/create_mantra_ifd.py

```diff
@@ -32,9 +32,13 @@ def create(self, product_name, instance_data, pre_create_data):
         parms = {
             # Render frame range
             "trange": 1,
-            # Arnold ROP settings
+            # Mantra ROP settings
             "soho_diskfile": filepath,
-            "soho_outputmode": 1
+            "soho_outputmode": 1,
+            # Enable `Save Geometery Inline`
+            # As it's hard to update the geo reference
+            # inside the IFD file after publishing the IFDs.
+            "vm_inlinestorage": 1
```

Comment:
Containing all the geometry into the output file means you're basically bloating the filepath a lot. I really feel like this is the opposite of what a 'render' file should look like - it should be a lightweight file whenever possible.

I understand the comment as to what is hard about changing paths in the output files - but I'm wondering, why would we need changing anyway? Is it because of mixed farms, etc?

Also, if this is a "local" submission - do we even need the IFD file? Or what's this about?

Aren't IFD's intermediate files just used for rendering? They don't need publishing - they are not proxies. They are just files used for the current render. If it's a local render - then technically it can almost live in memory or transparently be in a temp folder on the local machine?

Reply:
> why would we need changing anyway? Is it because of mixed farms, etc?

As far as I know it's not only about mixed farms but also about having the IFD files with the correct data. 
and keeping that link may produce IFD files that points to a wrong/non-existing geometry files. 

Also, I doubt this problem is going to occur also with farm IFD publishing. 

> Aren't IFD's intermediate files just used for rendering? They don't need publishing - they are not proxies. They are just files used for the current render.

I'm not sure about the `why we have IFD family` I can obly remember it was added by @moonyuet in https://github.com/ynput/OpenPype/pull/4903 
Could @moonyuet  you shed some light ?

Reply:
> Containing all the geometry into the output file means you're basically bloating the filepath a lot. I really feel like this is the opposite of what a 'render' file should look like - it should be a lightweight file whenever possible.
> 
> I understand the comment as to what is hard about changing paths in the output files - but I'm wondering, why would we need changing anyway? Is it because of mixed farms, etc?
> 
> Also, if this is a "local" submission - do we even need the IFD file? Or what's this about?
> 
> Aren't IFD's intermediate files just used for rendering? They don't need publishing - they are not proxies. They are just files used for the current render. If it's a local render - then technically it can almost live in memory or transparently be in a temp folder on the local machine?

We can just remove the entire product type instead. i mean same thing has been raised by @fabiaserra before